### ISSUE TITLE
Map and Document the user journey

### ISSUE THREAD

**Issue by pavlosts** [contribute-design/contribute.design#65]
It order to grow this project and make it easier for designers to contribute to OSS, we should understand and document the journey better. What steps do they take? What parts feel difficult to them? Where could improve the journey? etc.

Together with @Konstantinos019 we did a first draft of this journey but is based only on one user. It would be better to base our journey on more users to have a better overview. Here is what we have so far:

### First time user (Non-tech designer)

1. Homepage
2. For designers
3. Projects
4. Searches for project
    1. Not enough info for projects
    2. Info/description is not clear
    3. Information is very developer-oriented
5. Chooses a project
    1. Looks for an interesting project
    2. Doesn’t understand descriptions
    3. Can’t find out why they need a design contribution
    4. Feeling lost
    5. Project names are complicated
6. Picked one that looked small and had a more detailed description
7. Enters project page
8. Project website
9. Plays with project
10. Goes back to the project page
11. [DESIGN.md](http://design.md/)
12. Reads it (💭 “I have to know how to use GitHub”)
13. Tries to find an access point for open issues
14. Explores GitHub
15. Finds open issues
16. Doesn’t find design issues
17. Feels lost / doesn’t know what to do next
18. Thinking of making a design and posting it in GitHub or reaching out to someone but doesn’t know how
19. Almost gives up
20. Opens Figma and starts designing
21. Opens new issue
22. Didn't view design guidelines before starting design

**Comment by fivenp** [comment]
I think this comes back to how OSS projects document their design needs and how to educate them around preparing their guidelines. 

Maybe you want to try out the "For developers" flow and identify potential improvements there to solve the problems of designers? :) 

We do have a comprehensive set of "examples" -> https://github.com/contribute-design/examples
I'm sure there is also a lot of room to improve there....
